components/AdListItems/index.js

```diff
@@ -0,0 +1,87 @@
+import { data } from "../utils/data";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function AdListItems() {
+  data.sort((a, b) => {
+    if (a.createdDate > b.createdDate) {
+      return -1;
+    }
+    if (a.createdDate < b.createdDate) {
+      return 1;
+    }
+    return 0;
+  });
```

Comment:
Very good thought! If your sort function is as simple as this, you can condense everything to this:
```suggestion
  data.sort((a, b) => a .createdDate - b.createdDate);
```
This is merely a suggestion. You may leave it in its explicit form if you like.

Regardless, your code doesn't work yet because `createdDate` is of type `string`. But you need dates to do operations on. See the comment in the data.js for more info on how to do that.